packages/story-editor/src/components/checklist/karma/checklist.karma.js

```diff
@@ -398,63 +398,6 @@ describe('Checklist integration - Card visibility', () => {
     ACCESSIBILITY_COPY.videoMissingCaptions.title,
   ];
 
-  beforeEach(() => {
-    // mock the wordpress media explorer
```

Comment:
Why does this get removed?

Reply:
This is some leftover from when we decoupled the core editor from WordPress. There‘s no WP API that needs mocking in Karma tests.